.github/workflows/demo-testing.yml

```
@@ -34,3 +34,46 @@ jobs:
+          image_name: quay.io/konveyor/c-sharp-provider
+          image_tag: ${{ steps.set-tag.output.image_tag }}
+          dockerfile_path: Dockerfile
+          build_context: .
+
+      ## TODO: Once I have the artifact download working, this should run the analyzer-lsp from last working nightly based on branch.
+
+      - name: Run Provider test  
+        shell: bash
+        run: |
+          podman volume create test-data
+        podman run -rm -v test-data:/target:z -v $(PWD)/testdata:/src/:z --entrypoint=cp alpine -a /src/. /target/
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)